src/integrations/sdkinfo.ts

```diff
@@ -37,19 +48,14 @@ export class SdkInfo implements Integration {
       }
 
       event.platform = event.platform || 'javascript';
-      event.sdk = {
-        ...event.sdk,
-        name: SDK_NAME,
-        packages: [
-          ...((event.sdk && event.sdk.packages) || []),
-          ...((this._nativeSdkInfo && [this._nativeSdkInfo]) || []),
-          {
-            name: 'npm:@sentry/capacitor',
-            version: SDK_VERSION,
-          },
-        ],
-        version: SDK_VERSION,
-      };
+      event.sdk = event.sdk || {};
+      event.sdk.name = event.sdk.name || defaultSdkInfo.name;
+      event.sdk.version = event.sdk.version || defaultSdkInfo.version;
+      event.sdk.packages = [
+        // default packages are added by baseclient and should not be added here
```

Comment:
This doesn't apply to Capacitor. In RN it's true because we extend the `baseclient.ts` and fill the `options._metadata` with RN info. But Capacitor uses the browser client without changes. We should pass the `_metadata` to the browser client so that it can set them correctly, then this logic will work.

https://github.com/getsentry/sentry-react-native/blob/e0c46a6e00aa3d9edfe0a1301f1d527cacc89842/src/js/client.ts#L43-L44

Reply:
I added back the sentry/capacitor package to the project.